[Tue Feb 18 03:02:42.283855 2020] [authz_core:error] [pid 10644:tid 1980] [client 192.168.1.171:52835] AH01630: client denied by server configuration: Z:/xampp/phpMyAdmin/
[Sat Nov 21 09:55:30.850128 2020] [php7:error] [pid 5088:tid 1920] [client 127.0.0.1:51022] PHP Fatal error:  Uncaught PDOException: SQLSTATE[HY000] [1045] Access denied for user 'grupo77_repuesto'@'172.19.0.1' (using password: YES) in Z:\\_www\\api.carparts\\class\\Medoo\\Medoo.php:323\nStack trace:\n#0 Z:\\_www\\api.carparts\\class\\Apicore\\bin\\Core.php(90): Medoo\\Medoo->__construct(Array)\n#1 Z:\\_www\\api.carparts\\class\\Apicore\\bin\\Core.php(80): apicore\\bin\\core::startdb()\n#2 Z:\\_www\\api.carparts\\class\\Apicore\\ApiCore.php(20): apicore\\bin\\core->__construct()\n#3 Z:\\_www\\api.carparts\\class\\Jacwright\\RestServer\\RestServer.php(229): apicore\\apicore->__construct()\n#4 Z:\\_www\\api.carparts\\class\\Jacwright\\RestServer\\RestServer.php(138): Jacwright\\RestServer\\RestServer->instantiateClass('apicore\\\\apicore')\n#5 Z:\\_www\\api.carparts\\index.php(8): Jacwright\\RestServer\\RestServer->handle()\n#6 {main}\n  thrown in Z:\\_www\\api.carparts\\class\\Medoo\\Medoo.php on line 323
